functions/patchAPI.js

```diff
@@ -0,0 +1,34 @@
+const functions = require('firebase-functions');
+var cors = require('cors')({ origin: true });
+
+const admin = require('firebase-admin');
+
+const db = admin.firestore();
+
+exports.addCompletionDatePatch = functions.https.onRequest((request, response) => {
+    cors(request, response, () => {
+        console.log(request);
+        var completionDate = request.body.data.CompletionDate;
+        var id;
+        var result;
+        var p1 = db.collection("Tasks").get().then((querySnapshot) => {
```

Comment:
convert these into let and const type, according to the requirement, Do also use eslint for tracking these issue.

Reply:
Name querySnapshot is not good, please use something understandable.